handlers/createPlayer.go

```diff
@@ -10,6 +10,7 @@ import (
 
 type CreatePlayerResponse struct {
 	PlayerId string `json:"player_id"`
+	Game truco.Game `json:"game"`
 }
```

Comment:
@ch2ch3 - take a look at these `Response` structs. They'll be what will be marshalled into json and returned. Do they look right to you?